openslides_protocol/__init__.py

```diff
@@ -1,5 +1,7 @@
 __verbose_name__ = 'OpenSlides Protocol Plugin'
 __description__ = 'This plugin for OpenSlides provides a protocol of events managed in OpenSlides.'
 __version__ = '2.0-dev'
+__license__ = 'MIT'
+__url__ = 'http://openslides.org/'
```

Comment:
We should use the github url of this plugin (https://github.com/OpenSlides/openslides-protocol) to get more information in the new legal notice page. openslides.org url is already linked by OpenSlides core.